### PR TITLE
Integration Test - TagHelper Quick Info (Hover)

### PR DESCRIPTION
**Pull request by maryamariyan**
This PR adds an integration test for `TagHelper Quick Info` (hovering over a TagHelper).

Issue: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1861423/

### CODE REVIEW

File: test/razorIntegrationTests/hover.integration.tests.ts
Comment (davidwengier) [inline test/razorIntegrationTests/hover.integration.tests.ts:1]: From the CI logs, it doesn't look like the test actually ran. Perhaps try renaming the file to `hover.integration.test.ts` (note: no "s" after "test") to match the other filename. No idea is that will work, just a difference I noticed

Comment (maryamariyan) [reply test/razorIntegrationTests/hover.integration.tests.ts:1]: okay.

I also noticed this:

```
FAIL Razor Integration Tests test/razorIntegrationTests/formatting.integration.test.ts (9.379 s)


  ● Test suite failed to run

    thrown: "Token cancellation requested: {0}"

      at _getError (../../node_modules/jest-circus/build/utils.js:432:18)
          at Array.map (<anonymous>)

Test Suites: 1 failed, 1 total
```

File: test/razorIntegrationTests/hover.integration.test.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as path from 'path';
+import * as vscode from 'vscode';
+import * as jestLib from '@jest/globals';
+import testAssetWorkspace from './testAssets/testAssetWorkspace';
+import * as integrationHelpers from '../integrationTests/integrationHelpers';
+
+jestLib.describe(`Razor Hover ${testAssetWorkspace.description}`, function () {
```
Comment (dibarbet) [inline test/razorIntegrationTests/hover.integration.test.ts:12]: The `jestLib` is no longer needed - you can just import the functions normally now that we've removed mocha
```
import { describe, beforeAll, beforeEach, afterAll, test, expect } from '@jest/globals';
```

**Review by davidwengier — APPROVED** [review]
Nice work!

Techncially thats not a hover for a tag helper though, its just a html tag 😛

**Review by dibarbet — APPROVED** [review]

**Pull request merged**